src/plots/cartesian/graph_interact.js

```diff
@@ -309,6 +309,13 @@ fx.unhover = function (gd, evt, subplot) {
 // The actual implementation is here:
 
 function hover(gd, evt, subplot){
+    if(gd._fullLayout._hasPie){
```

Comment:
`evt` contains the data from the hover event - in this case, information on the slice that was hovered over. The `plotly_hover` event fires and gets passed data in the same format as the click events.